packages/client/src/lib.js

```diff
@@ -92,6 +92,18 @@ class NFTStorage {
       throw new Error(result.error.message)
     }
   }
+  /**
+   * @param {API.Service} service
+   * @param {Blob} blob
+   * @returns {Promise<API.CIDString>}
+   */
+  static async storeCar({ endpoint, token }, blob) {
+    const car =
+      blob.type !== 'application/car'
+        ? blob.slice(0, blob.size, 'application/car') // TODO: test this in safari.
```

Comment:
Seems to work

<img width="970" alt="Screenshot 2021-06-15 at 16 10 41" src="https://user-images.githubusercontent.com/152863/122078188-4ea3bd00-cdf4-11eb-9aba-5a2a2dc3445e.png">